src/layouts/sidebar-sidecar/utils/_local_platform-docs-mdx/components/enterprise-alert/index.tsx

```diff
@@ -16,34 +16,50 @@ import { isProductSlug, productSlugsToNames } from 'lib/products'
 import s from './enterprise-alert.module.css'
 
 interface EnterpriseAlertProps {
-	productSlug: ProductSlug
+	product: ProductSlug | 'hcp-vault-secrets' | 'hcp-vault-radar'
 	inline?: boolean
 	className?: string
 	children?: ReactNode
+	badgeText?: string
 }
 
 export function EnterpriseAlert({
-	productSlug,
+	product,
 	inline,
 	className,
 	children,
+	badgeText,
 }: EnterpriseAlertProps) {
 	// This ensures we aren't producing invalid HTML when rendering inline alerts within MDX. When used inline, we might end up nesting a div inside of a p. This is invalid as p cannot contain block-level elements (ref: https://www.w3.org/TR/html401/struct/text.html#h-9.3.1).
 	const Element = inline ? 'span' : 'div'
-	const isValidProduct = isProductSlug(productSlug)
+	const isVaultSecretsOrRadar =
+		product === 'hcp-vault-radar' || product === 'hcp-vault-secrets'
+	const isValidProduct = isProductSlug(product) || isVaultSecretsOrRadar
 
-	if (productSlug === 'hcp') {
+	if (product === 'hcp') {
 		return null
 	}
 
+	const productNames = {
+		...productSlugsToNames,
+		'hcp-vault-secrets': 'HCP Vault Secrets',
+		'hcp-vault-radar': 'HCP Vault Radar',
+	}
+
```

Comment:
If you wanted to, you could move this outside of the function so it's not re-created on every render, but it's not a blocker for merging this in by any means.